### ISSUE TITLE
Navigation - Inactive color

### ISSUE THREAD

**Issue by rileywho** [WikiEducationFoundation/WikiEduDashboard#332]
The current color for inactive items fails accessibility tests.

Please change the color to #696971

What it should look like:
![screen shot 2015-08-27 at 10 42 45 am](https://cloud.githubusercontent.com/assets/2120466/9528277/8ebe1f1c-4ca8-11e5-874f-57d8c8158396.png)
